### PR TITLE
Adds Rubik font v2.200

### PR DESCRIPTION
**Pull request by kuzzmi**
###### Motivation for this change

I am a huge fan of Rubik font and would like to add it to nixpkgs for everybody to use.

###### Things done

- [ ] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [X] NixOS
   - [ ] macOS
   - [X] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [X] Tested on a fresh NixOS installation both through Home Manager and from NixOS main configuration file
- [21.11 Release Notes (or backporting 21.05 Relase notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2111-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/data/fonts/rubik/rubik.nix [hunk 1]
```diff
@@ -0,0 +1,107 @@
+with import <nixpkgs> {};
```
Comment (SuperSandro2000) [inline pkgs/data/fonts/rubik/rubik.nix:1]:
```suggestion
{ stdenv, ... }:
```

File: pkgs/data/fonts/rubik/rubik.nix [hunk 2]
```diff
@@ -0,0 +1,107 @@
...
+
+  installPhase = ''
+    mkdir -p $out/share/fonts/truetype
+    cp -a *.ttf $out/share/fonts/truetype/
+  '';
+
+  outputHashAlgo = "sha256";
+  outputHashMode = "recursive";
+  outputHash = "0hd7969zdwzsyp5dsjlxhpg23byk1i05n2bb42sfr793pvc5g20d";
+
```
Comment (SuperSandro2000) [inline pkgs/data/fonts/rubik/rubik.nix:84]:
```suggestion
```

File: pkgs/data/fonts/rubik/rubik.nix [hunk 3]
```diff
@@ -0,0 +1,107 @@
...
+  meta = with lib; {
+    homepage = "https://fonts.google.com/specimen/Rubik";
+    description = "Rubik Fonts";
```
Comment (SuperSandro2000) [inline pkgs/data/fonts/rubik/rubik.nix:87]:
```suggestion
    description = "Rubik Font - is a 5 weight Roman + Italic family.";
```

**Review by SuperSandro2000 — CHANGES_REQUESTED** [review]
Please follow the contributing guide:
- name the commit ``rubik-font: init at 2.200``
- add an entry to top-level.nix
- do not reference the nixpkgs channel but instead add the inputs separately

**Comment by kuzzmi** [comment]
@SuperSandro2000 

I updated my commit with all three requested changes.  
Also, I added the font package to `top-level/all-packages.nix`, not sure tho if that is the one I had to update.

**Pull request merged**